.gitignore

```diff
@@ -77,6 +78,7 @@
 **/vendor/bundle/ruby/*/gems/mechanize-*/lib/mechanize/u*
 **/vendor/bundle/ruby/*/gems/mechanize-*/lib/mechanize/x*
 **/vendor/bundle/ruby/*/gems/thread_safe-*/lib/thread_safe/util
+**/vendor/bundle/ruby/*/gems/byebug-*/
```

Comment:
Oops looks like this hasn't been quite enough: https://github.com/Homebrew/brew/pull/7577/checks?check_run_id=685031720#step:20:63